src/backend/main.ts

```diff
@@ -1585,22 +1583,11 @@ ipcMain.handle('getFonts', async (event, reload) => {
 ipcMain.handle(
   'runWineCommandForGame',
   async (event, { appName, commandParts, runner }) => {
-    const gameSettings = await gameManagerMap[runner].getSettings(appName)
-
     if (isWindows) {
       return execAsync(commandParts.join(' '))
     }
-    const { updated } = await verifyWinePrefix(gameSettings)
 
-    if (runner === 'gog' && updated) {
-      await setup(appName)
-    }
-    if (runner === 'nile' && updated) {
-      await nileSetup(appName)
-    }
-    if (runner === 'legendary' && updated) {
-      await setupUbisoftConnect(appName)
-    }
+    await prepareWineLaunch(runner, appName)
```

Comment:
`runWineCommandOnGame` now already runs `prepareWineLaunch`. Wouldn't this run it twice?